test/performance/mako/sidecar.go

```diff
@@ -72,11 +74,18 @@ func Setup(ctx context.Context, extraTags ...string) (context.Context, *quicksto
 		return nil, nil, nil, err
 	}
 
+	// Get GCP project ID as a tag.
+	projectID, err := metadata.ProjectID()
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("project-id cannot be retrieved: %v", err)
```

Comment:
We shouldn't require things to run on a GCP-based cluster, but we should tag it with the project-id when it is.